flang/test/Fir/invalid.fir

```diff
@@ -541,6 +541,83 @@ fir.global internal @_QEmultiarray : !fir.array<32x32xi32> {
 
 // -----
 
+func @bad_save_result(%buffer : !fir.ref<!fir.array<?xf64>>, %n :index) {
```

Comment:
Aren't the test below this duplicating all the lit test above ?

https://github.com/flang-compiler/f18-llvm-project/blob/a2e1f114f2dff4075621719127a85d2cd06db628/flang/test/Fir/invalid.fir#L393-L457

Reply:
Right looks like I mixed my diff after a while. Sorry for the noise :-)